ttnn/cpp/ttnn/tensor/tensor_impl.hpp

```diff
@@ -275,7 +275,7 @@ inline void read_data_from_device_buffer(DeviceBuffer device_buffer, vector<T>&
 // ======================================================================================
 
 template <typename T>
-Tensor to_host(const Tensor& tensor, bool blocking = true);
+Tensor to_host(const Tensor& tensor, bool blocking = true, uint8_t cq_id = 0);
```

Comment:
Should we be using the same DefaultQueueId constant we use for ops for these other fns as well?

Reply:
done